packages/turbo/README.md

```diff
@@ -76,6 +74,59 @@ The library implements a native view `RNVisitable` component for React Native. T
 
 URL for the WKWebview to open. Changing the url should result in view replacing opening different page.
 
+### `sessionHandle`
+
+Session handle for the webview. If not provided, the default session will be used. It can be used to create separate webview instances for different parts of the app.
+
+### `stradaComponents`
+
+`VisitableView` supports defining [`Strada components`](https://strada.hotwired.dev/) - components that receive and reply to messages from web components that are present on the page _within one session_. This prop accepts an array of `Strada` components that will be registered in the webview.
```

Comment:
Basically, if you were to write the app natively on Android or iOS, you would need to define a global array of Strada component factories to use Strada. The user-agent is later updated based on this global array.

Now, if the user wants to define two VisitableViews with the same `sessionHandle` (one native WebView instance) but different `stradaComponents`, the `userAgent` would update properly, however, this change is not detected by Strada on the web for some reason, which should be investigated. This might be connected to the different approach used when writing the app natively, where the userAgent is set once (when initialising the native WebView). That's why I've decided to highlight "within one session".

Reply:
I'm not sure we want to support having multiple instances of `VisitableView` with the same `sessionHandle`. I can't think of a reason why you'd want that.

Reply:
Yeah, you're right, I also can't think of the use case. I think it would make sense to remove the emphasis from that sentence.